src/components/AssetDetailContent.jsx

```diff
@@ -85,10 +85,10 @@ const AssetDetailContent = (props) => {
                   <div><p>Asset Status</p></div>
                 </Grid.Column>
                 <Grid.Column className="details-description">
-                  <div><p>{assetDetail.asset_code}</p></div>
-                  <div><p>{assetDetail.serial_number}</p></div>
-                  <div><p>{assetDetail.model_number}</p></div>
-                  <div><p>{assetDetail.current_status}</p></div>
+                  <div><p>{assetDetail.asset_code ? assetDetail.asset_code : '-'}</p></div>
```

Comment:
Instead of using ternary operator here, use `||`. I.e
```
<p>{assetDetail.asset_code || '-'}</p>
```

This apply to all the changes in this file.

Please read about [Logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Logical).